src/appleseed/CMakeLists.txt

```diff
@@ -1632,6 +1632,8 @@ set (renderer_modeling_bsdf_sources
     renderer/modeling/bsdf/glassbsdf.h
     renderer/modeling/bsdf/glossybrdf.cpp
     renderer/modeling/bsdf/glossybrdf.h
+    renderer/modeling/bsdf/glossylayerbsdf.cpp
```

Comment:
I assume that not exposing this BSDF in the API header files is intentional?

Reply:
Yes. It only works when using OSL.